internal/webhooks/vmoperator/vspheremachine.go

```diff
@@ -0,0 +1,105 @@
+/*
+Copyright 2024 The Kubernetes Authors.
```

Comment:
Without changes to our webhook configurations this will never be called :)

(we'll need something in generate-manifests, and we need an appropriate output folder for supervisor webhooks (e.g. config/supervisor/webhook), eventually they have to end up in `infrastructure-components-supervisor.yaml`)

Reply:
Thanks for catching this! Just noticed `make generate-manifest`s only cover `./internal/webhooks` for now. Will update it to generate webhooks configs for supervisor mode.